src/agent/Core/ApplicationPool/Group/ProcessListManagement.cpp

```diff
@@ -81,8 +81,13 @@ Group::findProcessWithStickySessionIdOrLowestBusyness(unsigned int id) const {
 
 	if (leastBusyProcessIndex == -1) {
```

Comment:
I found a bug in the post-condition description for `findBestProcess()`:

```c++
@post result != nullptr || result.canBeRoutedTo()
```

This should be `@post result == nullptr || result.canBeRoutedTo()`, so can you fix that too?